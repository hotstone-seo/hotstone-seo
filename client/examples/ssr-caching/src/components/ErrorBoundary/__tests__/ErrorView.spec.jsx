import React from 'react';
import ErrorView from '../ErrorView.jsx';

test('Should render ErrorView correctly', () => {
  const tree = shallow(<ErrorView error={new Error('test')} info={{}}/>);
  
  expect(tree).toMatchSnapshot();
});
