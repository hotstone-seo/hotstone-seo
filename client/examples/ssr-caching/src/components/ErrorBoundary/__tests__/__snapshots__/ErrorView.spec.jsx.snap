// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ErrorView correctly 1`] = `
<BrowserRouter>
  <ErrorBoundary
    error={[Error: test]}
    info={Object {}}
  />
</BrowserRouter>
`;
